proto/buf.gen.ui.yaml

```diff
@@ -0,0 +1,10 @@
+version: v1
+managed:
+  enabled: true
+  go_package_prefix:
+    default: github.com/rilldata/rill
+plugins:
+  - name: es
+    path: ../node_modules/.bin/protoc-gen-es
```

Comment:
Lemme take a look.

Reply:
Found a better way to configure this. But we still need the npm package since it doesnt support remote package. Also we need the package while encode/decode proto in the browser.